### ISSUE TITLE
enable use of multiple parameters for the same distribution

### ISSUE THREAD

**Issue by papsti** [phac-nml-phrsd/ern#192]
for example, a user may have several fecal shedding distributions estimated from literature and they want to simulate by drawing one (with replacement) for each run. perhaps they want to weight the draws with some measure of confidence for each distribution?

**Comment by davidchampredon** [comment]
@papsti : i think this should be left out the package and dealt by the user.

**Comment by papsti** [comment]
agreed, but currently this wouldn't be easy because we only return the ensemble summary of Rt and not the individual Rt realizations. a user would need the latter so that they can go and make their own ensemble summary across distribution parameter sets.

perhaps we should include the individual realizations as one of the entries in the list returned by the `estimate_Rt_xx()` functions? this may be useful beyond the issue of passing multiple parameter sets for a specific distribution.
